R/dir_version.R

```diff
@@ -24,15 +24,15 @@ dir_version <- function(){
 #' }
 #' @export
 set_version <- function(version){
-	
+	.Deprecated(new = "shell script inst/mac_build.sh")
 	os = Sys.info()['sysname']
 	if (os == 'Darwin'){
 		current.pointer <- '/Library/Frameworks/R.framework/Versions/Current'
-		set.dir <- sprintf('/Library/Frameworks/R.framework/Versions/%s', version)
-		if (dir.exists(set.dir))
+		set.dir <- version
+		#if (dir.exists(set.dir))
 			system(sprintf('ln -sfhv %s %s', set.dir, current.pointer))
-		else 
-			warning(set.dir, ' does not exist.', call. = FALSE)
+		#else 
+			#warning(set.dir, ' does not exist.', call. = FALSE)
```

Comment:
why did you comment this out?

Reply:
oops, I was messing with things to try to make it work from Rstudio